eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -8,38 +8,34 @@ parameters:
   TimeoutInMinutes: 60
   PublishCodeCoverage: false
   Matrix:
-    Linux_Node10 (Public):
+    Linux_Node10:
       OSVmImage: "ubuntu-16.04"
       TestType: "node"
-      CloudType: public
       PublishCodeCoverage: true
-    Windows_Node10 (Public):
+    Windows_Node10:
       OSVmImage: "windows-2019"
       TestType: "node"
-      CloudType: public
-    macOS_Node10 (Public):
+    macOS_Node10:
       OSVmImage: "macOS-10.15"
       TestType: "node"
-      CloudType: public
-    Browser_Linux_Node10 (Public):
+    Browser_Linux_Node10:
       OSVmImage: "ubuntu-16.04"
       TestType: "browser"
-      CloudType: public
       PublishCodeCoverage: true
-    Browser_Windows_Node10 (Public):
+    Browser_Windows_Node10:
       OSVmImage: "windows-2019"
       TestType: "browser"
-      CloudType: public
-    Browser_macOS_Node10 (Public):
+    Browser_macOS_Node10:
       OSVmImage: "macOS-10.15"
       TestType: "browser"
-      CloudType: public
 
 jobs:
   - job: "IntegrationTest"
 
     variables:
       - template: ../variables/globals.yml
+      - name: CloudType
+        value: AzureCloud
```

Comment:
Why did you decide to do this variable here for JS but added it to each matrix entry in Java?

Reply:
The number of matrix entries across all of the services specified directly in the service `tests.yml` files in JS is rather high. Modifying each one (and then later modifying them to support other clouds) would be costly and repetitive. 

Another alternative is to put them into the template expansion: https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/jobs/archetype-sdk-integration.yml#L47

Reply:
OK. I'm all about sharing I was actually thinking we should default to AzureCloud in all languages and matrix entries and only specify it in the matrix entry if it isn't the default.